project/HmrcBuild.scala

```diff
@@ -40,13 +40,14 @@ object HmrcBuild extends Build {
 
 private object AppDependencies {
 
-  import play.PlayImport._
+  import play.sbt.PlayImport._
   import play.core.PlayVersion
 
   val compile = Seq(
     "com.typesafe.play" %% "play" % PlayVersion.current,
     ws,
-    "uk.gov.hmrc" %% "http-verbs" % "5.0.0"
+    "uk.gov.hmrc" %% "http-verbs" % "6.0.0",
+    "com.ning" % "async-http-client" % "1.8.15"
```

Comment:
it only in there to replace the Base64 library used in the DeviceFingerprint (which was previously transitively included via play 2.3).
I tried replacing from apache commons (which is currently transitively included) but that behaves differently - it doesn't fail on a junk string.
Version 1.8.15 was simply the one included by the play 2.3.